README.rst

```diff
@@ -1,340 +1,121 @@
-The async_generator library
-===========================
+.. image:: https://img.shields.io/badge/chat-join%20now-blue.svg
+   :target: https://gitter.im/python-trio/general
+   :alt: Join chatroom
+
+.. image:: https://img.shields.io/badge/docs-read%20now-blue.svg
+   :target: https://async-generator.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
 
 .. image:: https://travis-ci.org/njsmith/async_generator.svg?branch=master
    :target: https://travis-ci.org/njsmith/async_generator
    :alt: Automated test status
```

Comment:
Note that this image and the codecov one below link to njsmith/async_generator.

Reply:
Whoops, good catch. Want to fix it? :-)

Reply:
Sure, https://github.com/python-trio/async_generator/pull/10 :)